patches/node/build_use_third_party_simdutf.patch

```diff
@@ -0,0 +1,51 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Charles Kerr <charles@charleskerr.com>
+Date: Mon, 9 Dec 2024 11:18:51 -0600
+Subject: build: use third_party/simdutf
+
+use the Chromium version of simdutf to avoid duplicate symbols
```

Comment:
One thing I wasn't sure of is whether we should also delete the electron_node/deps/simduft [from this patch](https://github.com/electron/electron/blob/roller/chromium/main/patches/chromium/build_allow_electron_to_use_exec_script.patch#L33-L34) to `.gn` since we don't need this deps anymore

Reply:
It looks to me like it can be removed. I've pushed up an update of 177adfcb2bf1647ef0c730bf1ca1277d4bcb89ef that removes it :+1: